client/src/components/generic/CollapsibleList.jsx

```diff
@@ -1,13 +1,15 @@
 import React, { useState, useRef } from "react";
 import './CollapsibleList.scss'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCaretDown, faCaretUp  } from '@fortawesome/free-solid-svg-icons'

```

Comment:
Add this to App.jsx so you don't have to call it on each page that might need it.

Reply:
Awesome, didn't know about that.  Added to App.jsx.  Thanks!